src/routes/resources/[resourceContentId]/+page.svelte

```diff
@@ -519,6 +537,26 @@
         }
         return users?.filter((u) => $userIsInCompany(u.company.id)) ?? null;
     }
+
+    async function handleCommunityTranslate() {
+        if (!$currentUser) {
```

Comment:
I'm not sure we need this check, not that it hurts but $currentUser should always be set or we have bigger problems 😄

Reply:
yeah, this was because TS linter was saying it's possible to have `$currentUser` as `null`. Then, you cannot `newTranslationLanguageId = $currentUser?.languageId;` bc the definition of newTranslationLanguageId is number | undefined...not null. I don't like it either

Reply:
Ah interesting, one option then would be to do `newTranslationLanguageId = $currentUser?.languageId ?? null;` maybe? Nitpicking at this point but seeing the `!$currentUser` up there sends kind of a signal that it's an expected state when it shouldn't be. I probably need to take another look at addressing the underlying cause and making sure `$currentUser` is recognized as non-null .

Reply:
Opted for the `newTranslationLanguageId = $currentUser?.languageId ?? null;` syntax. Seems to satisfy the little red lines 😆